pagopa/io-eudiw-app
# chore: [EUDIW-153] Update presentation screen

## Short description
This PR updates the presentation screen aligning it with the [FIGMA](https://www.figma.com/design/IAq7xk1P2iBL1X9zdboMJk/EUDIW---Potential?node-id=1028-2499&node-type=frame&t=l9ABsO6SCmTVzeIm-0).

## List of changes proposed in this pull request
- Add `ItwMarkdown` component to render markdown;
- Add `ItwRequiredClaims` component to render the required claims for the presentation;
- Update `ItwIssuanceCredentialAuthScreen` to use the aforementioned components;
- Update locales;
- (Extra) Shows the `Reset` button only in debug;

## How to test
Try to obtain a credential and check that the screen matches the one in the FIGMA, except for the logos.
